workbenchfx-core/src/main/java/com/dlsc/workbenchfx/view/ToolbarPresenter.java

```diff
@@ -51,6 +52,11 @@ public ToolbarPresenter(Workbench model, ToolbarView view) {
     init();
     // Adds initially a menuButton if necessary (size of items > 0)
     setupMenuBtn();
+
+    // check initially whether to use the single module layout (size of modules = 1)
+    if (model.isSingleModuleLayout()) {
+      view.bottomBox.setVisible(false);
```

Comment:
just tested the changes and I noticed the `bottomBox` is still there, even after setting it to invisible. Can you please add `view.bottomBox.setManaged(...);` here and below, where you set the value to the same you set the visibility to? (so here it would be `false` as well for example).
This will not show the bar as well.